### ISSUE TITLE
AttributeError: 'Namespace' object has no attribute 'test'

### ISSUE THREAD

**Issue by gbicann** [icann/lgr-core#32]
I am running the `lgr_compare.py` script inside on Alpine Linux (using Docker, the Dockerfile is [here](https://gist.github.com/gbicann/c429f4feeacc3ced37aae1e6d0404bf4)).

When I try to compare the [Arabic language LGR for .abudhabi](https://www.iana.org/domains/idn-tables/tables/abudhabi_ar_4.0.xml) with the [reference second-level LGR for Arabic](https://www.icann.org/sites/default/files/packages/lgr/lgr-second-level-arabic-language-18may21-en.xml) I get this error:

```
/files # lgr_compare.py -1 lgr-second-level-arabic-language-18may21-en.xml -2 abudhabi_ar_4.0.xml DIFF
Traceback (most recent call last):
  File "/usr/local/bin/lgr_compare.py", line 4, in <module>
    __import__('pkg_resources').run_script('lgr-core==6.0.0', 'lgr_compare.py')
  File "/usr/local/lib/python3.11/site-packages/pkg_resources/__init__.py", line 672, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python3.11/site-packages/pkg_resources/__init__.py", line 1472, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python3.11/site-packages/lgr_core-6.0.0-py3.11.egg/EGG-INFO/scripts/lgr_compare.py", line 84, in <module>
    main()
  File "/usr/local/lib/python3.11/site-packages/lgr_core-6.0.0-py3.11.egg/EGG-INFO/scripts/lgr_compare.py", line 38, in main
    parser.setup_logger()
  File "/usr/local/lib/python3.11/site-packages/lgr_core-6.0.0-py3.11.egg/lgr/tools/utils.py", line 337, in setup_logger
    if self.args.test and not self.args.verbose:
       ^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'test'
```

**Comment by j-bernard** [comment]
Thanks for reporting.
The issue is targeted for version 6.0.1.
You can already test with the [v6.0.1 development branch](https://github.com/icann/lgr-core/tree/v6.0.1).

**Comment by j-bernard** [comment]
Fixed in v6.1.0
